opentelemetry-otlp/src/lib.rs

```diff
@@ -344,6 +346,19 @@ impl TonicPipelineBuilder {
         self
     }
 
+    /// Use `channel` as tonic's transport channel.
+    /// this will override tls config and should only be used
+    /// when working with non-HTTP transport.
```

Comment:
Nit: I think `transport` should be plural here?

Reply:
@TommyCpp I think he meant the other one, first line singular, last line plural

Reply:
Oops sorry I misunderstand that one 😬